package.json

```diff
@@ -13,11 +13,13 @@
   },
   "scripts": {
     "build": "rimraf dist && tsc",
+    "build:prepare": "[ -d \"dist/\" ] || npm run build",
```

Comment:
Getting `'[' is not recognized as an internal or external command, operable program or batch file.` using Windows CMD when running `npm install`

Reply:
Wooops, sorry, did it in zsh and didn't check for windows cmd. I've now switched to a cross-platform alternative.